### PR TITLE
Add first scene dates

### PR DESCRIPTION
**Pull request by ajd6735**
In this commit, I incorporated animated date objects, added facts modal to each, and adjusted the mission-facts-test file to ensure facts are now under 100 characters. It's worth noting that our documentation specifies the content should be less than 60 words for each fact, not 60 characters.

### CODE REVIEW

File: src/scene/scenes/AssemblyScene.tsx [hunk 1]
```diff
@@ -20,9 +23,12 @@ const AssemblyScene: SceneComponent = () => (
         <meshBasicMaterial transparent opacity={0} />
       </Box>
     </ExplodeTrigger>
-    <Orbiter position={[0, 2, 0]} scale={orbiterScale} />
-    <AssembleTestSceneName position={[0, 7, 2]} scale={sceneNameScale} />
+    <Orbiter position={[0, 6, -2]} scale={orbiterScale} />
+    <AssembleTestSceneName position={[0, 12, -1]} scale={sceneNameScale} />
     <AssembleAnimation />
+    <ModelOutliner color={0xffffff}>
```
Comment (zlix12) [inline src/scene/scenes/AssemblyScene.tsx:29]: `SceneManager` already wraps all of the in a `ModelOutliner` so you should not need it here.

File: src/artifacts/AssembTestDates.tsx
Comment (zlix12) [inline src/artifacts/AssembTestDates.tsx:1]: Seems like this is not used?

**Review by zlix12 — CHANGES_REQUESTED** [review]
Looks good visually, just a few fixes needed.

File: src/artifacts/AssembTestDates.tsx
Comment (ajd6735) [reply src/artifacts/AssembTestDates.tsx:1]: I admire your attention to detail, yes you are correct, this file is unnecessary.

**Review by zlix12 — APPROVED** [review]
LGTM

**Pull request merged**